cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.7
+EXPOSE 5775 6831 6832 14267 3000 3001 8080 8081 8082 8083
+
+ADD . /go/src/github.com/uber/jaeger
```

Comment:
1. building outside of the container is faster than inside it, which is especially useful for `npm install` which takes a long time. 
 1. this is a docker image we are going to publish and people are going to download. It makes no sense to have all the source files in that image. You *could* do the build and remove the sources, but then see (1). Also, the base image can be a lot smaller if we only need to put a binary in there, as opposed to having both Go and Node toolchains installed